### PR TITLE
VIH-6527 remove reference field from participant

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-6527

### Change description ###
Removed from participant representative Reference field

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite.AcceptanceTests/Steps/AddParticipantSteps.cs [hunk 1]
```diff
@@ -160,9 +160,6 @@ private void SetNewRepDetails(UserAccount user)
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(AddParticipantsPage.DisplayNameTextfield).SendKeys(user.DisplayName);
             var organisation = _c.Test.TestData.AddParticipant.Participant.Organisation;
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(AddParticipantsPage.RepOrganisationTextfield).SendKeys(organisation);
-            var reference = _c.Test.TestData.AddParticipant.Participant.Reference;
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(AddParticipantsPage.ReferenceTextfield).SendKeys(reference);
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(AddParticipantsPage.RepresentingTextfield).SendKeys(user.Representee);
```
Comment (npoxon) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Steps/AddParticipantSteps.cs:165]: Add this back in

Comment (npoxon) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Steps/AddParticipantSteps.cs:165]: AddParticipantsPage.RepresentingTextfield

File: AdminWebsite/AdminWebsite.AcceptanceTests/Steps/AddParticipantSteps.cs [hunk 2]
```diff
@@ -185,9 +182,6 @@ private void SetExistingRepDetails(UserAccount user)
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(AddParticipantsPage.DisplayNameTextfield).SendKeys(user.DisplayName);
             var organisation = _c.Test.TestData.AddParticipant.Participant.Organisation;
             EnterTextIfFieldIsNotPrePopulated(AddParticipantsPage.RepOrganisationTextfield, organisation);
-            var reference = _c.Test.TestData.AddParticipant.Participant.Reference;
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(AddParticipantsPage.ReferenceTextfield).SendKeys(reference);
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(AddParticipantsPage.RepresentingTextfield).SendKeys(user.Representee);
```
Comment (npoxon) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Steps/AddParticipantSteps.cs:190]: and here

File: AdminWebsite/AdminWebsite.AcceptanceTests/Steps/AddParticipantSteps.cs (hunk 1 above)
Comment (npoxon) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Steps/AddParticipantSteps.cs:165]: just the last line you have removed

**Review by npoxon — APPROVED** [review]

**Review by RocheL — APPROVED** [review]

**Pull request merged**